tools/cell_census_builder/source_assets.py

```diff
@@ -11,6 +11,10 @@
 from .mp import cpu_count, create_process_pool_executor
 
 
+def _get_n_workers() -> int:
```

Comment:
Required for testing. In GHA, running with more than 1 worker causes the test to fail.

Reply:
Can you point me at more information on this constraint?  I'm trying to understand _why_ it would fail on GHA.

If it is the case that GHA processes may not fork more than some limited number of child processes, we would be better off just testing this with the command line arg (`args` param) with a `--max-workers 1`.  This is the only code that calls `create_process_pool_executor` with the optional override, and it only affects the behavior when `args` lacks a `max_workers` value.

Reply:
The error is [here](https://github.com/chanzuckerberg/cell-census/actions/runs/4419756699/jobs/7748726289): 
```concurrent.futures.process.BrokenProcessPool: A process in the process pool was terminated abruptly while the future was running or pending.```
I didn't notice that the max_workers can be specified in args. Will use that. I'm not sure it's worth investigating the issue above - in my experience testing doesn't always play well with multiprocessing, so since this test isn't specifically meant to test multiprocessing, I think we can just disable it.

Reply:
Ah, this means something killed the child process.  Commonly this is an out-of-memory condition on the system, or something else that causes an uncontrolled failure (segv, etc).  Super hard to diagnose in that environment.